### DISCUSSION TITLE
Open Shell icon is behind the W11 icon

### DISCUSSION THREAD

**Discussion by karrdr** [Open-Shell/Open-Shell-Menu#978] [Q&A] [answered]
Hi
I have just installed W11 on the new laptop but Open Shell does not work as expected - the icon added by Open Shell collides with the original Win icon. Do you know how to fix it?

![screen](https://user-images.githubusercontent.com/100519061/155882664-0712c3fd-0564-4cbe-ad0b-fb6ff91c3fd6.png)

**Comment by MEdson71** [comment]
Hello!
This article I just found may be helpful.

https://www.askvg.com/tip-restore-old-classic-start-menu-in-windows-11/

**Comment by tallman69** [comment, marked as answer]
Before I installed Open Shell v4.4.169-beta on my Windows 11 laptop I installed ExplorerPatcher, and configured it to perform many of the tasks that ASKVG goes through in the above posted link and other tutorials he has done.  I did not have any trouble with the Open Shell start button covering the stock Windows 11 start button.  I have read a number of people have had the same issue you are having, but I believe that ExplorerPatcher fixes that automatically.  You can get ExplorerPatcher here:

https://github.com/valinet/ExplorerPatcher

**Comment by Win10nner** [comment]
Does it work on windows 10 22h2

**Reply by tallman69** [reply]
If you are asking about ExplorerPatcher, it is supposed to work on Windows 10, but they say some features may not be available.  I do not have any experience with ExplorerPatcher on windows 10 since most of what I use ExplorerPatcher for on Windows 11 is still available on Windows 10, so I don't use it on Windows 10.
